src/lib/viewers/doc/_docBase.scss

```diff
@@ -49,6 +49,23 @@ $thumbnail-border-radius: 4px;
     .bp-thumbnail.bp-thumbnail-is-selected .bp-thumbnail-nav {
         border-color: $fours;
     }
+
+    .bp-thumbnails-dark {
```

Comment:
May want to consider using BEM-ish syntax here to make it more clear that it's a modifier/override. Something like `.bp-thumbnail-nav--dark`?